_posts/2016/2016-01-15-setup-a-coreos-cluster-just-for-fun-on-centos-and-win.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: "Setting up a CoreOS Cluster just for fun on CentOS and Windows"
+date: "2015-12-12 17:17:17"
```

Comment:
If you can make the changes until tomorrow, then put 2016-01-15, if not, put the date of next Monday.